src/obb/server/http_component.clj

```diff
@@ -0,0 +1,39 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.http-component
+  "The aleph server as a component"
+  (:require [com.stuartsierra.component :as component]
+            [ring.middleware.cors :as cors]
+            [aleph.http :as http]
+            [compojure.core :as compojure]
+            [obb.server.routes :as routes]))
+
+(defn- setup-cors
+  "Setup cors"
+  [handler]
+  (cors/wrap-cors handler
+                  :access-control-allow-origin
+                  [#"^http://localhost(.*)"]
```

Comment:
Generally leaving for later... but this is just CORS, I usually just list on this collection all possible regexes (for example: localhost, *clanhr.com).